apps/pie-docs/src/engineers/getting-started/structure.md

```diff
@@ -49,6 +50,93 @@ For more details on Fozzie, Snacks and Skip PIE, please check out the **document
 
 ---
 
+## What library to use?
+
+### Latest tech stack and Vanilla JS
+
+{% contentLayout %}
+    {% tag {
+    label: "Next 14",
+    variant: "blue"
+    } %}
+    {% tag {
+    label: "React 18",
+    variant: "blue"
+    } %}
+    {% tag {
+    label: "Nuxt 3",
+    variant: "blue"
+    } %}
+    {% tag {
+    label: "Vue 3",
+    variant: "blue"
+    } %}
+    {% tag {
+    label: "Vanilla JS",
+    variant: "blue"
+    } %}
+{% endcontentLayout %}
+
+If you are working with latest stack both in React and Vue ecosystems, we recommend to go straight away with
+[PIE Web Components](/engineers/web-components/)
+
+### Next 13 and below 
+
+{% contentLayout %}
+    {% tag {
+    label: "Next 13",
+    variant: "blue"
+    } %}
+    {% tag {
+    label: "Next 10",
+    variant: "blue"
+    } %}
+{% endcontentLayout %}
+ 
+For Next 13 and below, we recommend using [PIE Web Components](/engineers/web-components/)
+only for client side components. If you need to render components on the server (SSR), [Snacks](https://snacks.takeaway.com/portal/) is the best library for you.
```

Comment:
I think it might be simpler and safer to recommend Snacks as the default library for React, but then say that PIE web components can be used for 13+ (CSR) or 14+ (SSR).

Reply:
I added an extra bit about updating to Next 14 if you would like to use Web Components. I hope with the tags saying `Next 13` and the title of the section also referring to Next 13 folks won't get confused :)